boards/arm/nrf52_adafruit_feather/nrf52_adafruit_feather.yaml

```diff
@@ -0,0 +1,7 @@
+identifier: nrf52_adafruit_feather
+name: nRF52 Adafruit Feather
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gccarmemb
```

Comment:
I know why CI is failing now. You need to add 2 lines here:
```
ram: 64
flash: 512
```
See https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/nrf52_pca10040/nrf52_pca10040.yaml